src/main/blackbox/blackbox.c

```diff
@@ -1050,6 +1050,9 @@ void blackboxValidateConfig(void)
 #endif
 #ifdef USE_SDCARD
     case BLACKBOX_DEVICE_SDCARD:
+#endif
+#if defined(SIMULATOR_BUILD) && defined(USE_BLACKBOX_VIRTUAL)
```

Comment:
USE_BLACKBOX_VIRTUAL already depends on SIMULATOR_BUILD so we can simply use
```suggestion
#if defined(USE_BLACKBOX_VIRTUAL)
```